ServerCore/Pages/Teams/TeamMergeSummary.cshtml.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using ServerCore.DataModel;
+using ServerCore.ModelBases;
+
+namespace ServerCore.Pages.Teams
+{
+    [Authorize(Policy = "IsEventAdmin")]
+    public class TeamMergeSummaryModel : EventSpecificPageModel
+    {
+        public TeamMergeSummaryModel(
+            PuzzleServerContext serverContext,
+            UserManager<IdentityUser> userManager)
+            : base(serverContext, userManager)
+        {
+        }
+
+        public class TeamSummary{
+            public TeamSummary(int teamId, string teamName, string mergedTeams)
+            {
+                TeamID = teamId;
+                TeamName = teamName;
+                MergedTeams = mergedTeams; 
```

Comment:
I would consider decoding here but no biggie

Reply:
Recorded in #855